src/views/ipv/pyi-triage-select-device.njk

```diff
@@ -0,0 +1,43 @@
+{% extends "shared/base.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% set pageTitleName = 'pages.pyiTriageSelectDevice.title' | translate %}
+{% set googleTagManagerPageId = "pyiTriageSelectDevice" %}
+
+{% set errorState = pageErrorState %}
+{% set errorTitle = 'pages.pyiTriageSelectDevice.content.formErrorMessage.errorSummaryTitleText' | translate %}
+{% set errorText = 'pages.pyiTriageSelectDevice.content.formErrorMessage.errorSummaryDescriptionText' | translate %}
+{% set errorHref = "#pyiTriageSelectDeviceForm" %}
+
+{% block content %}
+  <h1 class="govuk-heading-l" id="header" data-page="{{ googleTagManagerPageId }}">{{ 'pages.pyiTriageSelectDevice.header' | translate }}</h1>
+  <p class="govuk-body">{{ 'pages.pyiTriageSelectDevice.content.paragraph1' | translate }}</p>
+
+  <form id="pyiTriageSelectDeviceForm" action="/ipv/page/{{ pageId }}" method="POST" onsubmit="return pyiTriageSelectDeviceFormSubmit()">
```

Comment:
I don't think we need the `onsubmit` handler here anymore.

I believe we previously would add in this on each page so that we prevent the user from double-submitting the form by pressing continue twice.

But we now have a global script that should reproduce this behaviour automatically.

@blakeyp is that correct?

Reply:
Relatedly `pyiTriageSelectDeviceFormSubmit()` doesn't actually appear to be defined anywhere?

Reply:
The script Roger mentions appears to be in `init.js` which gets built into `application.js` which is included in `base.njk`